cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/replication/BasicPullStrategy.java

```diff
@@ -374,8 +377,6 @@ private ChangesResultWrapper nextBatch() throws DatastoreException {
     public Iterable<DocumentRevsList> createTask(List<String> ids,
                                                         Map<String, Collection<String>> revisions) {
```

Comment:
Obviously not part of your original work in this PR but these method arguments don't line up.